Diagnostics/HealthChecker/Tests/HealthChecker.Tests.ps1

```diff
@@ -338,6 +338,14 @@ Describe "Testing Health Checker by Mock Data Imports" {
             SetActiveDisplayGrouping "Security Settings"
             TestObjectMatch "AMSI Enabled" "False" -WriteType "Yellow"
         }
+
+        It "EEMS Enabled And OCS Reachable" {
```

Comment:
Was the `Mock` added to import the XML data for this already?

Reply:
Yes: `Mock Get-Service { return Import-Clixml "$Script:MockDataCollectionRoot\Exchange\GetServiceMitigation.xml" }`.
Telemetry data is in organization configuration which is also available. Pattern service check is performed in context of the machine which runs the unit tests so outbound internet connectivity is possible (we receive a 200 OK).